modules/reducers/utils.js

```diff
@@ -30,6 +30,11 @@ export const computePlan = ({ payload }) => {
 		daysLeft: payload.tier_validity
 			? Math.ceil((payload.tier_validity - new Date().getTime() / 1000) / (24 * 60 * 60))
 			: 0,
+		clusterDaysLeft: payload.cluster_tier_validity
```

Comment:
@ShahAnuj2610 here we are already getting number of days so you don't need to calculate days here

Reply:
days for the cluster are different than that for the app, it's `tier_validity` vs `cluster_tier_validity`

Reply:
Right but if you check response here, https://i.imgur.com/46jASGy.png

`cluster_tier_validity` is already a number of days remaining for trial, so you don't need to `(payload.cluster_tier_validity - new Date().getTime() / 1000) / (24 * 60 * 60)`

Reply:
oh, I see but shouldn't the datatypes for both be same?

Reply:
Yes datatype would be same, but its just that you don't need to calculate